.travis.yml

```diff
@@ -112,6 +121,12 @@ jobs:
     - script: make nightly-release
       env: PACKAGE_TYPE=deb RESTY_IMAGE_BASE=ubuntu RESTY_IMAGE_TAG=bionic KONG_PACKAGE_NAME=${PWD##*/} KONG_VERSION=`date +%Y-%m-%d` REPOSITORY_NAME=${PWD##*/}-nightly REPOSITORY_OS_NAME=$TRAVIS_BRANCH
       if: type=cron
+    - script: make nightly-release
```

Comment:
Fun side note: I love how the stage is called "deploy daily build" but the command is "make nightly release" :joy: